### PR TITLE
fix(deps): replace alpha versions with release versions

### PR DESCRIPTION
**Pull request by phiz71**
## Issue

N/A

## Description

Prepare to merge on 4.6.x branch

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-jchezexomk.chromatic.com)

### CODE REVIEW

**Review by jgiovaresco — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -53,16 +53,16 @@
         <gravitee-cloud-initializer.version>2.1.0</gravitee-cloud-initializer.version>
         <gravitee-spec-gen-api.version>1.1.0</gravitee-spec-gen-api.version>
         <gravitee-common.version>4.6.0</gravitee-common.version>
-        <gravitee-secret-api.version>1.0.0-alpha.3</gravitee-secret-api.version>
+        <gravitee-secret-api.version>1.0.0</gravitee-secret-api.version>
```
Comment (graviteeio) [inline pom.xml:56]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secret-api.version>*****</gravitee-secret-api.version>  
[This comment was created by Aqua Pipeline]

File: pom.xml [hunk 2]
```diff
@@ -263,10 +263,10 @@
         <gravitee-resource-auth-provider-http.version>1.4.0</gravitee-resource-auth-provider-http.version>
         <gravitee-resource-auth-provider-inline.version>1.4.0</gravitee-resource-auth-provider-inline.version>
         <gravitee-resource-auth-provider-ldap.version>1.4.0</gravitee-resource-auth-provider-ldap.version>
-        <gravitee-resource-cache-redis.version>3.0.0-alpha.1</gravitee-resource-cache-redis.version>
+        <gravitee-resource-cache-redis.version>3.0.0</gravitee-resource-cache-redis.version>
         <gravitee-resource-oauth2-provider-keycloak.version>2.1.0</gravitee-resource-oauth2-provider-keycloak.version>
         <gravitee-service-geoip.version>3.0.0</gravitee-service-geoip.version>
-        <gravitee-secretprovider-kubernetes.version>2.0.0-alpha.2</gravitee-secretprovider-kubernetes.version>
+        <gravitee-secretprovider-kubernetes.version>2.0.0</gravitee-secretprovider-kubernetes.version>
```
Comment (graviteeio) [inline pom.xml:269]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:** etprovider-kubernetes.version>*****</gravitee-secretpro  
[This comment was created by Aqua Pipeline]

File: pom.xml [hunk 3]
```diff
@@ -282,14 +282,13 @@
         <gravitee-policy-graphql-rate-limit.version>1.0.1</gravitee-policy-graphql-rate-limit.version>
         <gravitee-resource-schema-registry-confluent.version>3.1.0</gravitee-resource-schema-registry-confluent.version>
         <gravitee-reactor-message.version>5.0.0-alpha.11</gravitee-reactor-message.version>
-        <gravitee-reactor-native-kafka.version>1.0.0-alpha.37</gravitee-reactor-native-kafka.version>
+        <gravitee-reactor-native-kafka.version>1.0.0-alpha.38</gravitee-reactor-native-kafka.version>
         <gravitee-apim-repository-bridge.version>5.1.2</gravitee-apim-repository-bridge.version>
-        <gravitee-secretprovider-hc-vault.version>2.0.0-alpha.4</gravitee-secretprovider-hc-vault.version>
-        <gravitee-secretprovider-aws.version>2.0.0-alpha.2</gravitee-secretprovider-aws.version>
-        <gravitee-service-secrets.version>1.0.0-alpha.4</gravitee-service-secrets.version>
+        <gravitee-secretprovider-hc-vault.version>2.0.0</gravitee-secretprovider-hc-vault.version>
```
Comment (graviteeio) [inline pom.xml:287]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secretprovider-hc-vault.version>*****</gravitee-secretprovider-hc-vault.version>  
[This comment was created by Aqua Pipeline]

File: pom.xml [hunk 4]
```diff
@@ -282,14 +282,13 @@
...
+        <gravitee-secretprovider-aws.version>2.0.0</gravitee-secretprovider-aws.version>
```
Comment (graviteeio) [inline pom.xml:288]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secretprovider-aws.version>*****</gravitee-secretprovider-aws.version>  
[This comment was created by Aqua Pipeline]

File: pom.xml [hunk 5]
```diff
@@ -282,14 +282,13 @@
...
+        <gravitee-service-secrets.version>1.0.0</gravitee-service-secrets.version>
```
Comment (graviteeio) [inline pom.xml:289]: ### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-service-secrets.version>*****</gravitee-service-secrets.version>  
[This comment was created by Aqua Pipeline]

**Pull request merged**